openshift/pkg/otp/testdata/networking/network_segmentation/udn/udn_test_pod_readiness_template.yaml

```
@@ -0,0 +1,34 @@
+metadata:
+  name: udn-pod-template
+objects:
+- kind: Pod
+  apiVersion: v1
+  metadata:
+    name: "${NAME}"
+    namespace: "${NAMESPACE}"
+    labels:
+      name: "${LABEL}"
+  spec:
+  spec:
```

YAMLlint (1.38.0)

[error] duplication of key "spec" in mapping

(key-duplicates)